bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/api/OpenSprinklerHttpApiV100.java

```diff
@@ -242,4 +261,91 @@ private JcResponse statusInfo() throws CommunicationApiException {
         public List<List<Integer>> ps;
         public int rs;
     }
+
+    /**
+     * This class contains static methods for communicating HTTP GET
```

Comment:
```suggestion
     * This class contains helper methods for communicating HTTP GET
```

Can you move that to a own class-file?`I assume you use that to simplify the communication (instead of calling the jetty methods directly), right?

Reply:
Yes, that's the intention. I'm not sure, if it makes sense to move it to it's own class file. This would expose the Http class to other users of, at least, the package, however, it is meant to be strongly tight to the http api implementation. That's why it's an inner class of the Http Api implementation. Does this make sense? :)

Reply:
Ok. But I would prefer a better name. Maybe HttpHelper or OpenSprinklerHttp or something which sounds more specialized than Http.

Reply:
Ok, I choosed "HttpRequestSender", as this describes what it is doing :)